### DISCUSSION TITLE
Unable to resolve dependency [Parameter #0 [ <required> $id ]]

### DISCUSSION THREAD

**Discussion by abramaleko** [livewire/livewire#3670] [1. Help] [answered]
This weird i am trying to send a id to the verify method it works locally and in production shows an error of Unable to resolve dependency.

Here is the code which trigger the event 

`  <a wire:click='verifyComment({{$comment->id}})' class="px-4 py-2 text-white bg-indigo-600 rounded cursor-pointer hover:bg-indigo-900">Verify</a>
`

And the function 

```
public function verifyComment($id)
    {
       $comment=Comments::find($id);
       $comment->verified=true;
       $comment->save();
       //updates the dashboard
       $this->getComments();
       $this->getTrendingPosts();

       session()->flash('message', 'Comment successfully verified.');
    }
```

I have tried  so many using events but still can not make it work in production

**Comment by jonneroelofs** [comment, marked as answer]
try wire:click="verifyComment('{{$comment->id}}')"
